rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -260,12 +235,12 @@ namespace
 			mip = D3D12_FILTER_TYPE_LINEAR;
 			return;
 		case CELL_GCM_TEXTURE_CONVOLUTION_MIN:
-			LOG_WARNING(RSX, "CELL_GCM_TEXTURE_CONVOLUTION_MIN not supported, fallback to bilinear filtering");
 			min = D3D12_FILTER_TYPE_LINEAR;
 			mip = D3D12_FILTER_TYPE_POINT;
 			return;
 		}
-		unreachable("Wrong min filter");
+		unreachable("Unimplemented min filter used %x, please report this to a developer.", min_filter);
+
```

Comment:
@tambry please, stay civilized here, no bullying =)
First part of O1L suggestion is too short and doesn't urge the user to create issue on github (report to dev) while second part is cryptic/blah to be correctly interpreted.

I can see nothing wrong with detailed log message as long as it fits in one line.